app/src/main/java/com/ranggacikal/challengechapter5/ui/LeaderBoardFragment.kt

```diff
@@ -43,16 +36,22 @@ class LeaderBoardFragment : Fragment(), HistoryView {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        presenter = HistoryPresenter(this)
-        presenter.getListHistory(requireContext())
+
+        sharedPreference = PreferenceHelper
+        val prefs = sharedPreference.customPreference(requireContext(), CUSTOM_PREF_NAME)
+
+        binding.rvHistory.layoutManager = LinearLayoutManager(requireContext(), RecyclerView.VERTICAL, false)
```

Comment:
Change Layout Manager to 

binding.rvHistory.layoutManager = GridLayoutManager(context,1)